From f270f80d4c36a22a0f7b8de8fb6b1abef6f7b183 Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Tue, 15 Feb 2022 16:23:10 +0800
Subject: [PATCH] plymouth : Add the retain-splash option
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Adding the "retain-splash” option to tell plymouth to keep the boot
splash’s contents on screen even after plymouth exits to make the
splash screen show more friendly on the system which boots so fast.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
---
 systemd-units/plymouth-quit.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/systemd-units/plymouth-quit.service.in b/systemd-units/plymouth-quit.service.in
index ed9f9b9..147e9ee 100644
--- a/systemd-units/plymouth-quit.service.in
+++ b/systemd-units/plymouth-quit.service.in
@@ -3,7 +3,7 @@ Description=Terminate Plymouth Boot Screen
 After=rc-local.service plymouth-start.service systemd-user-sessions.service
 
 [Service]
-ExecStart=-@PLYMOUTH_CLIENT_DIR@/plymouth quit
+ExecStart=-@PLYMOUTH_CLIENT_DIR@/plymouth quit --retain-splash
 Type=oneshot
 RemainAfterExit=yes
 TimeoutSec=20
-- 
2.17.1

